src/fontra/views/editor/editor.js

```diff
@@ -873,7 +874,10 @@ export class EditorController {
     };
   }
 
-  setSelectedTool(toolIdentifier) {
+  setSelectedTool(toolIdentifier, setToolIdentifier = true) {
```

Comment:
This new argument is not needed if we rearrange the logic a little bit.

I think the space-down and space-up code should be themselves be responsible for remembering which tool was selected. I will comment more below.

In the meantime, `this.selectedToolIdentifier` should be set at the end of this method, right after `this.sceneController.setSelectedTool(this.tools[selectedToolIdentifier]);`, and it should use the `selectedToolIdentifier` variable.